src/frame.rs

```diff
@@ -5,9 +5,7 @@ use errors::{check, check_not_null, check_success, Error};
 use property::{PropertiesIter, Property, RawProperty};
 use strings;
 
-use super::{Atom, AtomMut, AtomRef};
-use super::{BondOrder, Residue, Topology, TopologyRef};
-use super::{UnitCell, UnitCellMut, UnitCellRef};
+use super::{Atom, AtomMut, AtomRef, BondOrder, Residue, Topology, TopologyRef, UnitCell, UnitCellMut, UnitCellRef};
```

Comment:
Same here, the separate imports are intentional

Reply:
Sure thing, let me revert those changes.